test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -205,6 +215,7 @@ async function setup(t) {
 test
   .requestHooks(sendEmailMock)('send email screen should have right labels', async t => {
     const challengeEmailPageObject = await setup(t);
+    await t.expect(challengeEmailPageObject.formExists()).eql(true);
```

Comment:
Should this be added to the `setup` function under the `navigateToPage` function call?